policies/committer-policy.md

```diff
@@ -103,7 +103,6 @@ agree that a submission is trivial and the _cla: trivial_ label should
 be applied to indicate this.
 
 [Contributor Agreements]: https://www.openssl.org/policies/cla.html
-[OpenSSL Bylaws]: https://www.openssl.org/policies/omc-bylaws.html
```

Comment:
I don't think this one should be removed, but rather the github link on line 112...  and even more so since this is the link used in the glossary...

Reply:
The point is that the link here goes to the glossary which points to the canonical place.  This way, changing something's location involves updated one link not many.

The following links are all to the glossary not to the definition.  Why should this one be special?

Reply:
Ah ok, good point.